pkg/binutils/binaries.go

```diff
@@ -309,7 +309,9 @@ func cleanupPluginDir(vmID, pluginDir string) error {
 
 	pluginWhiteList := map[string]struct{}{
 		"evm": {},
-		vmID:  {},
+	}
+	for vmID := range vmIDs {
```

Comment:
I think it is not good practice to modify an input param. It cames from Download() then intro cleanupDir(). Modifying it may introduce problems. Better not touching it.

Reply:
Ah you are very right here, I thought for a moment that it was a copy of the object, but golang does indeed change the origin object when it's a map.